src/CommunityToolkit.Maui/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -245,15 +245,13 @@ protected override void OnDetachingFrom(VisualElement bindable)
 	}
 
 	/// <inheritdoc/>
-	protected override async void OnViewPropertyChanged(object? sender, PropertyChangedEventArgs e)
+	protected override async void OnViewPropertyChanged(VisualElement sender, PropertyChangedEventArgs e)
```

Comment:
Can we use interface here?

Reply:
For Behaviors, we can't constrain to interfaces because `public IList<Behavior> Behaviors` isn't defined in any interface; it is only defined in `VisualElement`:

https://github.com/dotnet/maui/blob/ec33802c4ab453e13a24b1fc63bb019cd28152f9/src/Controls/src/Core/VisualElement.cs#L421-L424

This means that for `BaseBehavior<TView>`, the lowest that `TView` can be constrained is to `VisualElement`:
```cs
public abstract class BaseBehavior<TView> : Behavior<TView> where TView : VisualElement
```

And since `ValidationBehavior ` implements `BaseBehavior<TView>`, the lowest we can constrain `TView` is `InputView`:

```cs
public class ValidationBehavior : BaseBehavior<VisualElement>
```